tests/code/test_fileHandler.py

```diff
@@ -2,69 +2,130 @@
 import os
 from werkzeug.datastructures import FileStorage
 
+
+
 from app import app
+from app.models import mainDB
+from dateutil import parser
+from app.models.user import User
+from flask import g
+from app.logic.events import attemptSaveEvent
+
+
+from app.models.program import Program
 from app.models.attachmentUpload import AttachmentUpload
 from app.models.event import Event
 from app.logic.fileHandler import FileHandler
 
-# test events
+# creates base object for events tests
 eventFileStorageObject= [FileStorage(filename= "eventfile.pdf")]
 
 handledEventFile = FileHandler(eventFileStorageObject, eventId=15)
 
-# test course
+handledEventFileRecurring = FileHandler(eventFileStorageObject, eventId=16)
+
+# creates base objects for course tests
 courseFileStorageObject= [FileStorage(filename= "coursefile.pdf")]
 
 handledCourseFile = FileHandler(courseFileStorageObject, courseId=1)
 @pytest.mark.integration
 def test_getFileFullPath():
     # test event
-    filePath = handledEventFile.getFileFullPath(eventFileStorageObject[0])
+    filePath = handledEventFile.getFileFullPath("15/" + "".join(eventFileStorageObject[0].filename))
     assert filePath == 'app/static/files/eventattachments/15/eventfile.pdf'
     
     # test course
-    filePath = handledCourseFile.getFileFullPath(courseFileStorageObject[0])
+    filePath = handledCourseFile.getFileFullPath("".join(courseFileStorageObject[0].filename))
```

Comment:
Why are we using `join` here? We don't have multiple filenames, it's just one value.